src/goto-instrument/loop_utils.cpp

```diff
@@ -107,3 +108,41 @@ void get_modifies(
     }
   }
 }
+
+// TODO handle aliasing at all
```

Comment:
:-\  This makes me a little nervous...

Reply:
I'm working on removing this --- this function was introduced very briefly before it was removed and I thought that it hadn't made it into any of the pull requests. Next revision shouldn't have it (or any of the problems that came with it being very temporary).